src/reporters/base-reporter.js

```diff
@@ -170,6 +172,14 @@ export default class BaseReporter {
     }
   }
 
+  logNetworkUsage(bytes: number) {
```

Comment:
Why is this indirection needed if people can access `this.networkUsage` directly?